docs/changelog.md

```diff
@@ -136,6 +136,13 @@ Users who do not use GraphBinary yet or who are not using Geoshapes are not affe
 
 We are dropping support for old serialization format of JanusGraph predicates. The old predicates serialization format is only used by client older than 0.6. 
 The change only affects Gryo and GraphSON.
+##### Upgrade of log4j to version 2
+
+This change requires a new log4j configuration. You can find an example configuration in `conf/log4j2-server.xml`. As fact of the changed configuration format, 
+we cleanup all configurations. This could lead to unexpect new log lines. Please open an issue, if you have see any unwanted log line.
```

Comment:
```suggestion
we clean up all configurations. This could lead to unexpected new log lines. Please open an issue, if you see any unwanted log line.
```